### ISSUE TITLE
Add support for Python 3.11

### ISSUE THREAD

**Issue by mbrancato** [GoogleCloudPlatform/cloud-debug-python#90]
Docs currently say this supports versions 3.6-3.10 - there is no support or wheel for Python 3.11.

**Comment by mctavish** [comment]
That is correct.  3.11 uses a compressed linetable format that isn't supported by the current codebase.  It will take some time and effort to get the agent to work with that more complex format.

**Comment by mgrdcm** [comment]
I assume that due to #92 this won't get done :)

**Comment by mctavish** [comment]
Truth be told, I took a swing at it and ran into an unexpected issue with a segfault during expression evaluation.  If I can find some time I'm going to give it another try before archiving this repo but I can't promise anything.

**Comment by mgrdcm** [comment]
Ooh exciting!  Thanks for your work on that!

**Comment by mctavish** [comment]
Unfortunately I'm going to have to close this issue without completing it.  There are some issues with the agent's expression evaluation code in Python 3.11 that result in a reproducable segfault.  I don't have the bandwidth to dig further into this, so will be archiving the repo with the last supported version being 3.10.

I also looked into Python 3.12.  While PEP 669 looked like it could be promising and could simplify the Debugger's breakpoint implementation, it does not actually provide any improvements.  The Debugger would need to register for LINE events which would have significant performance impacts on the running application to the point that it would not be a workable solution.
